lib/tasks/backfills.rake

```diff
@@ -1,11 +1,17 @@
 desc "These tasks are meant to be run once then removed"
 namespace :backfills do
-  task backfill_conversation_unread_status: :environment do
-    Conversation.where(user_with_unread_messages: nil).includes(:messages).each do |conversation|
-      last_notification = conversation.messages.last.notifications_as_message.last
-      if last_notification&.unread?
-        conversation.update(user_with_unread_messages: last_notification.recipient)
+  task developer_response_rate: :environment do
+    begin
+      ActiveRecord::Migration.check_pending!
+    rescue ActiveRecord::PendingMigrationError => e
+      if e.message.include?("add_response_rate_to_developers")
+        puts e.message
+        exit 1
```

Comment:
Let's remove this and let the job fail on its own when the migration hasn't been run.

Reply:
Got it. TIL, thanks!